### PR TITLE
Update with 2024 options

### PR DESCRIPTION
**Pull request by rtesra**
Model options updated:

- Using country specific options from approved 2023/2024 Naomi outputs with the following updates/ defaults:
  - Calendar quarters  T2 - T5 updated to 2024 - 2027
  - ANC T2 options set to 2025

    - include_art_t1: blank (force select)
    - include_art_t2: TRUE
    - artattend: TRUE
    - artattend_t2: TRUE
  - Calibration method set to logistic

Note: For 2023 CAF, CMR and STP did not submit approved Naomi files. MWI and ZAF used bespoke model options. Default options from prior year were used in these cases.

### CODE REVIEW

**Comment by r-ash** [comment]
Testing with Naomi here https://github.com/mrc-ide/naomi/pull/447

**Review by r-ash — APPROVED** [review]
Looks good to me, passing Naomi tests nicely

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# naomi.options 1.3.0
+
+* Update default for 2024/2025 HIV estimates
```
Comment (jeffeaton) [inline NEWS.md:3]: Can you add to this the details of specific updates that you put in the PR notes? That may be helpful to refer back to next year.

**Review by jeffeaton — COMMENTED** [review]
Many thanks for this. 

> MWI and ZAF used bespoke model options. Default options from prior year were used in these cases.

To confirm, for these cases, you still updated the years to one year forward, correct?

File: inst/default_options.csv [hunk 2]
```diff
@@ -1,40 +1,41 @@
...
+MLI,MLI,1,CY2024Q4,CY2025Q3,CY2013Q1,MLI2012DHS,,,FALSE,,TRUE,TRUE,TRUE,2024,12,,2024,,2024,national,TRUE,250,1000,1000,TRUE,-4,TRUE,1,TRUE,1,TRUE,1,national,sex_age_group,national,sex_age_group,national,sex_age_group,national,sex_age_group,logistic,1,TRUE,TRUE,TRUE,FALSE,FALSE,TRUE,CY2026Q3,CY2027Q3,1
+MOZ,MOZ,3,CY2024Q4,CY2025Q3,CY2021Q3,MOZ2021PHIA,MOZ2021PHIA,,FALSE,,TRUE,TRUE,TRUE,2024,12,,2024,2021,2024,national,TRUE,250,1000,1000,TRUE,-4,TRUE,1,TRUE,1,TRUE,1,national,sex_age_group,national,sex_age_group,national,sex_age_group,national,sex_age_group,logistic,1,TRUE,TRUE,TRUE,FALSE,TRUE,TRUE,CY2026Q3,CY2027Q3,3
+MWI,MWI,5,CY2024Q4,CY2025Q3,CY2020Q3,MWI2020PHIA,MWI2020PHIA,,FALSE,,TRUE,TRUE,TRUE,2024,12,2020,2024,2020,2024,national,TRUE,250,1000,1000,TRUE,-4,TRUE,1,TRUE,1,TRUE,1,national,sex_age_group,national,sex_age_group,national,sex_age_group,national,sex_age_group,logistic,1,TRUE,TRUE,TRUE,FALSE,FALSE,TRUE,CY2026Q3,CY2027Q3,3
```
Comment (r-ash) [inline inst/default_options.csv:26]: Should this be 4 for the area level?

Test data is coming through with
* Country - level 0
* Region - level 1
* Zone - level 2
* District - level 3
* District + Metro - level 4

Comment (r-ash) [reply inst/default_options.csv:26]: No, we don't want to do this. The actual Malawi data does have 6 levels. So 5 is right

**Pull request merged**